packages/mobile/src/components/sign-out-confirmation-drawer/SignOutConfirmationDrawer.tsx

```diff
@@ -0,0 +1,75 @@
+import { useCallback } from 'react'
+
+import { signOut } from 'audius-client/src/common/store/sign-out/slice'
+import { setVisibility } from 'audius-client/src/common/store/ui/modals/slice'
+import { View } from 'react-native'
+
+import { Button, Text } from 'app/components/core'
+import { AppDrawer } from 'app/components/drawer/AppDrawer'
+import { useDispatchWeb } from 'app/hooks/useDispatchWeb'
+import { makeStyles } from 'app/styles'
+
+const MODAL_NAME = 'SignOutConfirmation'
+
+const messages = {
+  drawerTitle: 'HOLD UP!',
+  cancelText: 'NEVERMIND',
```

Comment:
Can we use `textTransform: 'uppercase'` instead of capitalizing the string itself?

Reply:
I tried this before and it didn't work, but then realized that you have to put it on the text styles of the button, not the button styles 🤦